README.md

```diff
@@ -86,40 +81,38 @@ with:
           td: 1
           hr: 0
         multipliers:
-          - targets: [ ISSUE, ISSUER, SPECIFICATION ]
+          - select: [ ISSUE_SPECIFICATION ]
             formattingMultiplier: 1
             wordValue: 0.1
-          - targets: [ ISSUE, ISSUER, COMMENTED ]
+          - select: [ ISSUE_AUTHOR ]
             formattingMultiplier: 1
             wordValue: 0.2
-          - targets: [ ISSUE, ASSIGNEE, COMMENTED ]
+          - select: [ ISSUE_ASSIGNEE ]
             formattingMultiplier: 0
             wordValue: 0
-          - targets: [ ISSUE, COLLABORATOR, COMMENTED ]
+          - select: [ ISSUE_COLLABORATOR ]
             formattingMultiplier: 1
             wordValue: 0.1
-          - targets: [ ISSUE, CONTRIBUTOR, COMMENTED ]
+          - select: [ ISSUE_CONTRIBUTOR ]
             formattingMultiplier: 0.25
             wordValue: 0.1
-          - targets: [ REVIEW, ISSUER, TASK ]
+          - select: [ PULL_SPECIFICATION ]
             formattingMultiplier: 0
             wordValue: 0
-          - targets: [ REVIEW, ISSUER, COMMENTED ]
+          - select: [ PULL_AUTHOR ]
             formattingMultiplier: 2
             wordValue: 0.2
-          - targets: [ REVIEW, ASSIGNEE, COMMENTED ]
+          - select: [ PULL_ASSIGNEE ]
             formattingMultiplier: 1
             wordValue: 0.1
-          - targets: [ REVIEW, COLLABORATOR, COMMENTED ]
+          - select: [ PULL_COLLABORATOR ]
             formattingMultiplier: 1
             wordValue: 0.1
-          - targets: [ REVIEW, CONTRIBUTOR, COMMENTED ]
+          - select: [ PULL_CONTRIBUTOR ]
```

Comment:
@gentlementlegen I need a refresher:
1. What's the difference between `ISSUE_COLLABORATOR` and `ISSUE_CONTRIBUTOR`?
2. What's the difference between `PULL_COLLABORATOR` and `PULL_CONTRIBUTOR`?

Collaborators are simply added to the organization?

Reply:
Collaborators are part of the organization or the repo, contributors are external users.
Issues are comments inside of the issue, and pull are comments inside of the pull-request.